devman

```diff
@@ -218,7 +218,7 @@ class DevMan:
             except Exception as e:
               print "Can't clone {0}: {1}".format(repo_name, e)
           else:
-            print 'Error: Repository %s does not exist. Please use git clone first.' % repo_name
+            print 'Error: Repository %s doesn\'t exist. Please use git clone first.' % repo_name
```

Comment:
@svigneux Damn my Python is rusty. Simple double-quotes would be enough in this case though.